.github/actions/publish-pypi/action.yml

```diff
@@ -17,7 +17,11 @@ runs:
       uses: actions/download-artifact@v3
       with:
         name: ${{ inputs.archive-name }}
-        path: .
```

Comment:
We should leave the distributions in the expected directory. Also added a log statement to see what's going on in this step.